.circleci/config.yml

```diff
@@ -93,14 +93,15 @@ workflows:
           {}
 
       - "nixos":
-          name: "<<matrix.pythonVersion>>"
-          nixpkgs: "nixpkgs-unstable"
+          name: "<<matrix.nixpkgs>>-<<matrix.pythonVersion>>"
```

Comment:
please include "nix" or similar in the name

Reply:
In fact, the new `nixpkgs` parameter introduced here should always start with `nixpkgs-` (convention from the `flake.nix`).

![image](https://github.com/user-attachments/assets/c5dcac32-edc0-4196-ba4f-9b1abf994183)

So we should not need an other `nix(os)?-` prefix.